migration/src/main/java/uk/ac/bbsrc/tgac/miso/migration/destination/DefaultMigrationTarget.java

```diff
@@ -180,6 +180,7 @@ private void saveSample(Sample sample) throws IOException {
       // already saved
       return;
     }
+    log.debug("Saving sample " + sample.getAlias());
```

Comment:
Actually, should move this below the next block because saving this sample will not happen until after saving the parent (if necessary)